src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/Elite2ApiService.java

```diff
@@ -68,7 +68,8 @@ public String getOffenderNoFromBookingId(final Long bookingId) {
     public List<Long> getBookingIdsForScheduleActivitiesByDateRange(final String prisonId, final TimePeriod period, final LocalDate fromDate, final LocalDate toDate) {
         final var url = "/schedules/{prisonId}/activities-by-date-range?fromDate={fromDate}&toDate={toDate}&timeSlot={period}";
 
-        final var responseType = new ParameterizedTypeReference<List<Map>>() {};
+        final var responseType = new ParameterizedTypeReference<List<Map>>() {
+        };
```

Comment:
Definitely not a blocker but looks like your IDE has been a bit overzealous again.

Reply:
I need to look into it but I suspect that's the standard Java way to format code with curly braces.

Reply:
Concise braces in Java are usually considered acceptable: https://google.github.io/styleguide/javaguide.html#s4.1.3-braces-empty-blocks